plugins/inputs/opcua/opcua.go

```diff
@@ -439,7 +454,7 @@ func (o *OpcUA) getData() error {
 	resp, err := o.client.Read(o.req)
 	if err != nil {
 		o.ReadError.Incr(1)
-		return fmt.Errorf("RegisterNodes Read failed: %v", err)
+		return fmt.Errorf("Read failed: %v", err)
```

Comment:
While refactoring, please also use `%w` for errors:
```suggestion
		return fmt.Errorf("Read failed: %w", err)
```